modules/machineset-modifying.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// 
+// * machine_management/modifying-machineset.adoc
+
+[id="machineset-modifying_{context}"]
+= Modifying a MachineSet
+
+When making changes to a MachineSet, you must remove all machines associated with the MachineSet 
+by deleting or scaling down the machines to `0`. Then scale the replicas back to the desired number. 
```

Comment:
We do not have to delete the machines, we just need to scale down the MachineSet. Deleting a machine will cause a new machine being created, if its replicas does not change.

Reply:
Both are fine. New machine replacing deleted one will be created from updated template.

Reply:
> Does this show that the user needs to edit the YAML first, then remove the machines?

It's better phrased. lg. thanks.